readme.md

```diff
@@ -1,16 +1,30 @@
-# WeePickle [![Build Status](https://travis-ci.org/rallyhealth/weePickle.svg)](https://travis-ci.org/rallyhealth/weePickle) <img src="https://api.bintray.com/packages/rallyhealth/maven/weePickle/images/download.svg" title="Download" />
+# weePickle [![Build Status](https://travis-ci.org/rallyhealth/weePickle.svg)](https://travis-ci.org/rallyhealth/weePickle) [![sbt instructions](https://api.bintray.com/packages/rallyhealth/maven/weePickle/images/download.svg)](#sbt)
 
-A JSON, YAML, MsgPack, XML, etc. serialization framework with MiMa and shading.
+A *stable* JSON, YAML, MsgPack, XML, etc. serialization framework based on [uPickle](https://lihaoyi.github.io/upickle).
 
-Safe for use in libraries without causing dependency hell ([rationale](shading.md)).
+## We're tired of dependency hell!
+weePickle exists to fulfill two promises:
+
+1. We will not break compatibility in `v1.x.y`, as enforced by [MiMa](https://github.com/lightbend/mima).
+2. When we release `v2.0.0`, you can use it immediately **without waiting for your other library dependencies to update**. We achieve this by [shading](https://github.com/rallyhealth/sbt-shading).
+
+### Shading
+Both `weepickle-v1.jar` and `weepickle-v2.jar` (in the future) will coexist on the classpath peacefully by applying [shading](https://github.com/rallyhealth/sbt-shading) at multiple levels.
+1. All artifact names are suffixed with `-v1`, which prevents evictions.
+2. All packages are prefixed with `com.rallyhealth.v1`, which prevents classpath conflicts.
```

Comment:
waffling between keeping that and replacing with `vX` since you mention v2 (and supposedly other versions) in the future. up to you.